controllers/challenge.js

```diff
@@ -12,10 +12,11 @@ const sendChallengeResponse = async (req, res) => {
   try {
     if (req.method === 'GET') {
       const allChallenges = await challengeQuery.fetchChallenges()
-      if (allChallenges.length > 0) {
+      const challengesWithParticipants = await getParticipantsofChallenges(allChallenges)
+      if (challengesWithParticipants) {
```

Comment:
If there are no challenges, that isn't the user's fault, right? 4xx are user's fault. We should be showing 200 only if challenges are not found with a message that no challenges found